downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -593,6 +593,10 @@ public function main() returns error? {
 
 - Specified SSL as the preferred option when users provide options without SSL configuration. Additionally, introduced support for explicitly disabling SSL.
 
+#### `java.jms` package
+
+- Introduced revamped JMS protocol connector for Ballerina.
```

Comment:
Shall we add what we have improved/revamped instead? probably we can add mention about the observability support, and improved documentation with examples. WDYT?

Reply:
Updated with [6e1012e94167d71da6e3a2281bbf643515c4d4fa](https://github.com/ballerina-platform/ballerina-dev-website/pull/8972/commits/6e1012e94167d71da6e3a2281bbf643515c4d4fa)